pkgs/applications/networking/instant-messengers/webcord/default.nix

```diff
@@ -56,36 +53,34 @@ buildNpmPackage rec {
     install -Dm644 sources/assets/icons/app.png $out/share/icons/hicolor/256x256/apps/webcord.png
 
     # Add xdg-utils to path via suffix, per PR #181171
-    makeWrapper '${electron_25}/bin/electron' $out/bin/webcord \
-      --prefix LD_LIBRARY_PATH : ${libPath}:$out/opt/webcord \
+    makeWrapper '${lib.getExe electron_25}' $out/bin/webcord \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libpulseaudio pipewire ]}:$out/opt/webcord \
       --suffix PATH : "${lib.makeBinPath [ xdg-utils ]}" \
       --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}" \
       --add-flags $out/lib/node_modules/webcord/
 
     runHook postInstall
   '';
 
-  desktopItems = [
-    (makeDesktopItem {
-      name = "webcord";
-      exec = "webcord";
-      icon = "webcord";
-      desktopName = "WebCord";
-      comment = meta.description;
-      categories = [ "Network" "InstantMessaging" ];
-    })
-  ];
+  desktopItems = makeDesktopItem {
+    name = "webcord";
+    exec = "webcord";
+    icon = "webcord";
+    desktopName = "WebCord";
+    comment = "A Discord and SpaceBar electron-based client implemented without Discord API";
```

Comment:
Is there a reason why it was changed from `meta.description` to the same string repeated?

Reply:
My personal hatred of `rec`, I reverted it